src/component/Header/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Header.scss';
+import './HeaderResponsive.scss';
+
+function Header() {

```

Comment:
Please use named export instead of `default export`.

Reply:
@MAslanweb

Reply:
@MAslanweb Talking about this.